src/tmx/TmxUtils/src/telemetry/TelemetrySerializer.h

```diff
@@ -0,0 +1,68 @@
+#pragma once
+#include <vector>
+#include <map>
+#include <jsoncpp/json/json.h>
+#include <boost/algorithm/string.hpp>
+
+#include <tmx/messages/message.hpp>
+
+#include "PluginTelemetry.h"
+#include "TelemetrySerializerException.h"
+#include "TelemetryHeader.h"
+#include "TelemetryMetadata.h"
+
+namespace tmx::utils::telemetry{
+    class TelemetrySerializer
```

Comment:
Same comment as before about using the serializer terminology. As I think about this, is this not more of a data type in that it is a collection of Telemetry objects? Then this class could be pretty synonymous with the PluginTelemetry class just as a vector of objects right?

Reply:
I guess this also puts the information in a message_container